openbb_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -809,13 +739,59 @@ def call_metric(self, other_args: List[str]):
             default=False,
             help="Output all raw data",
         )
-
+        parser.add_argument(
+            "--show-all",
+            dest="show_all",
+            action="store_true",
+            default=False,
+            help="Show all available metrics",
+        )
         if other_args and "-" not in other_args[0][0]:
             other_args.insert(0, "-m")
         ns_parser = self.parse_known_args_and_warn(
             parser, other_args, EXPORT_BOTH_RAW_DATA_AND_FIGURES
         )
         if ns_parser:
+
+            if ns_parser.show_all:
```

Comment:
Why adding a new flag? Why just not having this by default if no argument is provided? It's simpler and faster 😄

Reply:
I prefer having the user ask for it instead of just getting it, but I changed back just 4 u

Reply:
Just because it's faster!